src/components/Mobile/MobileProject/MobileProject.scss

```diff
@@ -1,25 +1,41 @@
+@use "../../../spacing" as *;
 @import "./../../../colours.scss";
 
 .proj-container--mobile {
   margin: 0;
 }
 
-.mobile-nav {
+#app .proj-container--mobile .react-tabs {
```

Comment:
❓ what's `#app` needed for across this stylesheet?

Reply:
It's to override the styles in `tabs.scss`, which may have needed `#app` to override the default styles from the library (if I remember correctly)

Reply:
Not sure whether it should be in `tabs.scss` though, as it's mainly overrides for the bottom tabs on mobile